examples/example-auto-scroll-when-dragging.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
+  <link rel="shortcut icon" type="image/ico" href="favicon.ico" />
+  <title>SlickGrid example: Spreadsheet with Excel compatible cut and paste</title>
+  <link rel="stylesheet" href="../slick.grid.css" type="text/css"/>
+  <link rel="stylesheet" href="../css/smoothness/jquery-ui.css" type="text/css"/>
+  <link rel="stylesheet" href="examples.css" type="text/css"/>
+  <style>
+    .cell-effort-driven {
+      text-align: center;
+    }
+    .slick-group-title[level='0'] {
+      font-weight: bold;
+    }
+    .slick-group-title[level='1'] {
+      text-decoration: underline;
+    }
+    .slick-group-title[level='2'] {
+      font-style: italic;
+    }
+    .slick-row .slick-cell.frozen:last-child, 
+    .slick-header-column.frozen:last-child, 
+    .slick-headerrow-column.frozen:last-child,
+    .slick-footerrow-column.frozen:last-child {
+      border-right: 1px solid red;
+    }
+
+    .slick-row.frozen:last-child .slick-cell {
+      border-bottom: 1px solid red;
+    }
+    .option-item {
+      padding: 6px;
+    }
+  </style>
+</head>
+<body>
+<div style="position:relative">
+  <div style="width:600px;">
+    <div id="myGrid" style="width:100%;height:350px;"></div>
+  </div>
+  <br/>
+
+  <div class="options-panel">
+    <h2>Options:</h2>
+    <div class="option-item">
+      <label for='isAutoScroll'>Auto Scroll: (need click Set Options)</label>
+      <input type="checkbox" id="isAutoScroll">
+    </div>
+
+    <div class="option-item">
+      <label for='minIntervalToShowNextCell'>MIN interval to show next cell: </label>
+      <input type='text' id="minIntervalToShowNextCell" style="width:60px;">
+      (ms)
+    </div>
+
+    <div class="option-item">
+      <label for='maxIntervalToShowNextCell'>MAX delay to show next cell (ms): </label>
+      <input type='text' id="maxIntervalToShowNextCell" style="width:60px;">
+      (ms)
+      <div>better to multiple of MIN</div>
+    </div>
+
+
+    <div class="option-item">
+      <label for='accelerateInterval'>Delay when cursor outside 1px (ms): </label>
+      <input type='text' id="accelerateInterval" style="width:60px;">
+      (ms)
+    </div>
+
+    <div class="option-item">
+      <button id='setOptions' onclick="setOptions()">Set Options</button>
+      <button id='setDefaultOption' onclick="setDefaultOption()">Set Default Options</button>
+    </div>
+
+    <hr>
+
+    <div class="option-item">
+      <button id='toggleFrozen' onclick="toggleFrozen()">Set/Clear Frozen</button>
+      <button id='toggleGroup' onclick="toggleGroup()">Set/Clear grouping by Duration</button>
+    </div>
+
+    <h2>Demonstrates:</h2>
+    <ul>
+      <li>Dragging to make a selection, and the viewport will auto scroll.</li>
```

Comment:
at the beginning I thought it wasn't working because I always started my selection from first column which is not selectable, can you add "...starting from second column..." somewhere there?

Reply:
You are right. I will add such message on the page